### ISSUE TITLE
crashes with ReferenceError: jQuery is not defined

### ISSUE THREAD

**Issue by chaseWilliams** [cockpit-project/cockpit-machines#596]
If Node.js isn't installed on the host machine, the installation still works just fine. However, the Virtual Machines tab fails to load and throws an "oops" on the top. 

Screenshot of console logs:
<img width="475" alt="Screen Shot 2022-02-24 at 6 28 52 PM" src="https://user-images.githubusercontent.com/14338864/155636258-1704abc7-9f87-43f1-8bdd-917d0bd345f2.png">
 
I fixed this by installing node, but nothing in the error messages or documentation (so far as I could tell) said anything about Node being a prerequisite.

**Comment by martinpitt** [comment]
Node really is *not* a dependency of cockpit-machines. The problem here is the "jQuery is not defined". Apparently you are running an old cockpit-machines version against a newer cockpit? Please tell us your distribution and the package versions.

**Comment by SpencerCornish** [comment]
:wave: @martinpitt  Not OP, but I just hit this too after following the installation guide for Ubuntu 20.04.

After installing node and reinstalling cockpit and cockpit-machines, everything worked. **But** I figured out a better solution!

When I installed cockpit, I did 
` sudo apt install -t ${UBUNTU_CODENAME}-backports cockpit` 

When I installed cockpit-machines, I did
`sudo apt install cockpit-machines`
which installed v215 (e.g. old).

By running `sudo apt install -t ${UBUNTU_CODENAME}-backports cockpit cockpit-machines` v260-1 was installed, and everything worked without Node installed (I removed it since it was huge)

## TLDR

 `sudo apt install -t ${UBUNTU_CODENAME}-backports cockpit-machines` fixes the problems without installing Node

**Comment by KKoukiou** [comment]
@chaseWilliams this is a known issue, has been reported in the past like in https://github.com/cockpit-project/cockpit-machines/issues/565

Please use backports to update the package.
